### DISCUSSION TITLE
Upgrade to Monolog 3.0

### DISCUSSION THREAD

**Discussion by carbontwelve** [laravel/framework#42753] [Ideas] [answered]
I'm not sure of the benefit of upgrading to 3.0 over the 2.0 branch of [monolog/monolog](https://github.com/Seldaek/monolog) given that both are actively developed, however recently I wanted to use [itspire/monolog-loki](https://github.com/itspire/monolog-loki) and have had to install an older version of that library with support for monolog 2.0 as its author has now switched over to 3.0.

```shell
% sail composer why monolog/monolog
laravel/framework        v9.17.0  requires  monolog/monolog (^2.0)        
spatie/ignition          1.3.1    requires  monolog/monolog (^2.0)        
spatie/laravel-ignition  1.2.4    requires  monolog/monolog (^2.3) 
```

**Comment by b-t-927** [comment]
I'm in the exact same boat with this library, would be good to be able to use the latest version

**Comment by crynobone** [comment]
Hey there,

We're closing this issue because it's inactive, already solved, old, or not relevant anymore. Feel free to open up a new issue if you're still experiencing this problem.

**Comment by shaedrich** [comment, marked as answer]
This was done via #42311
